src/UiPath.Workflow/Activities/ScriptingJitCompiler.cs

```diff
@@ -73,7 +74,12 @@ public override LambdaExpression CompileExpression(ExpressionToCompile expressio
                 .Select(GetTypeName));
         var finalCompilation = compilation.ReplaceSyntaxTree(syntaxTree, syntaxTree.WithChangedText(SourceText.From(
             CreateExpressionCode(types, names, expressionToCompile.Code))));
-        var results = ScriptingAotCompiler.BuildAssembly(finalCompilation);
+        
+        var collectibleAlc = new AssemblyLoadContext("ScriptingAot" + Guid.NewGuid(), true);
```

Comment:
I can also hide this under an internal "feature flag" so we can only enable this in Studio (for validate in designers). Let me know if this is preferable. 

For AOT compilation, we still use the standard assembly load context.